### PR TITLE
Export dataset.resolve field 

### PR DESCRIPTION
**Pull request by jbothma**

### CODE REVIEW

File: zavod/zavod/tests/exporters/test_exporters.py [hunk 1]
```diff
@@ -182,6 +182,7 @@ def test_ftm_referents(testdataset1: Dataset):
     identifier = resolver.decide(
         "osv-john-doe", "osv-johnny-does", Judgement.POSITIVE, user="test"
     )
+    testdataset1.resolve = True
```
Comment (jbothma) [inline zavod/zavod/tests/exporters/test_exporters.py:185]: not sure if this is just too yuck

**Review by pudo — CHANGES_REQUESTED** [review]
Can we make sure that the default for absent `resolve` is true? We also probably don't want to include it in the JSON unless it's false.

**Comment by jbothma** [comment]
Thanks, fixed.

File: zavod/zavod/tests/fixtures/testdataset1/testdataset1.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ entry_point: testentrypoint1
 title: OpenSanctions Validation Dataset
 prefix: osv
 hidden: true
+resolve: false
```
Comment (pudo) [inline zavod/zavod/tests/fixtures/testdataset1/testdataset1.yml:5]: This hits me as super surprising - that our main fixture is going to have one of the default features disabled. Does it not break any tests?

Comment (jbothma) [reply zavod/zavod/tests/fixtures/testdataset1/testdataset1.yml:5]: yeah it definitely has a smell, and not a nice one.

Only two failed!

I need to go, but I'll fix when I'm back. This is not an urgent change

**Pull request merged**